cs/mineralpacket_test.go

```diff
@@ -134,3 +134,45 @@ func TestMineralPacket_getPacketDecayRate(t *testing.T) {
 		})
 	}
 }
+
+func TestMineralPacket_estimateDamage(t *testing.T) {
+	type fields struct {
+		SafeWarpSpeed int
+		WarpSpeed     int
+	}
+	type args struct {
+		race              *Race
+		planetDriverSpeed int
+		planetPosition    Vector
+		planetDefCoverage float64
+		targetRace        *Race
+		planetPop         int
+		mass              Cargo
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   MineralPacketDamage
+	}{
+		{"1 yr away; no decay/dmg", fields{WarpSpeed: 5, SafeWarpSpeed: 5}, args{NewRace().WithSpec(&rules), 5, Vector{25.0, 0.0}, 0.0, NewRace().WithSpec(&rules), 1000000, Cargo{10, 10, 10, 0}}, MineralPacketDamage{}},
```

Comment:
This is a bit hard to know what each arg is, and you don't need `.0` on the floats since they are already declared as floats. Can you update the args to have named args?

```suggestion
		{
			"1 yr away; no decay/dmg",
			fields{WarpSpeed: 5, SafeWarpSpeed: 5},
			args{
				race:              NewRace().WithSpec(&rules),
				planetDriverSpeed: 5,
				planetPosition:    Vector{25, 0},
				planetDefCoverage: 0,
				targetRace:        NewRace().WithSpec(&rules),
				planetPop:         1000000,
				mass:              Cargo{10, 10, 10, 0},
			},
			MineralPacketDamage{},
		},
```